src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseService.cs

```diff
@@ -665,23 +712,20 @@ private IList<MethodologyVersion> GetMethodologiesScheduledWithRelease(Guid rele
     public class DeleteDataFilePlan
     {
         [JsonIgnore]
-        public Guid ReleaseId { get; set; }
+        public Guid ReleaseId { get; init; }
 
         [JsonIgnore]
-        public Guid SubjectId { get; set; }
+        public Guid SubjectId { get; init; }
 
-        public DeleteDataBlockPlan DeleteDataBlockPlan { get; set; } = null!;
+        public DeleteDataBlockPlan DeleteDataBlockPlan { get; init; } = null!;
 
-        public List<Guid> FootnoteIds { get; set; } = null!;
-    }
+        public List<Guid> FootnoteIds { get; init; } = null!;
 
-    public class DeleteReleasePlan
-    {
-        public IReadOnlyList<IdTitleViewModel> ScheduledMethodologies { get; init; } = [];
+        public bool IsLinkedToApiDataSetVersion { get; init; }
 
-        public IReadOnlyList<DeleteApiDataSetVersionPlan> ApiDataSetVersions { get; init; } = [];
+        public DeleteApiDataSetVersionPlan? LinkedApiDataSetVersionDeletionPlan { get; init; }
 
-        public bool Valid => ApiDataSetVersions.All(dsv => dsv.Valid);
+        public bool Valid => !IsLinkedToApiDataSetVersion;
```

Comment:
This is a bit confusing as the `LinkedApiDataSetVersionDeletionPlan` can display `Valid` as true, whilst the overall `Valid` is false.

It'd be more intuitive for `LinkedApiDataSetVersionDeletionPlan.Valid` to be settable to false to more clearly indicate the reason why it can't be deleted. Consequently, we should also be able to remove `IsLinkedToApiDataSetVersion` as this wouldn't be needed anymore./